backend/tests/test_auth_api.py

```
@@ -0,0 +1,27 @@
+from user_management.models import User
+def test_register(client, db_session):
+    # Sample data to register a new user
+    user_data = {
+        "username": "testuser03",
+        "email": "testuser03@example.com",
+        "password": "testpassword",
+        "full_name": "Test User",
+        "address": "123 Test Street",
+        "phone": "1234567890",
+        "role": "customer"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)